doc/meetings/build-wg-meeting-2022-01-25.md

```diff
@@ -0,0 +1,73 @@
+# Node.js  Build WorkGroup Meeting 2022-01-25
+
+## Links
+
+* **Recording**:  http://www.youtube.com/watch?v=7JbJp2kRCXg
+* **GitHub Issue**: https://github.com/nodejs/build/issues/2852
+
+## Present
+
+* Michael Dawson (@mhdawson)
+* Richard Lau (@richardlau)
+* Rich Trott (@trott)
+
+## Agenda
+
+## Announcements
+
+*Extracted from **build-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+
+### nodejs/build
+
+
+* Platform requirements for Node.js 18 [#2815](https://github.com/nodejs/build/issues/2815)
+  * Planned to last until ~ April 2025, so looking at what platforms which still be in support
+    at that time
+  * Main change is that centos 7 will EOL before then so planning to move to RHEL 8
+  * Applying for no cost RHEL subscription from Open Source program. 
```

Comment:
We could link to https://www.redhat.com/en/blog/extending-no-cost-red-hat-enterprise-linux-open-source-organizations

```suggestion
  * Applying for no cost [RHEL subscription for Open Source Infrastructure](https://www.redhat.com/en/blog/extending-no-cost-red-hat-enterprise-linux-open-source-organizations) program. 
```